browser/src/control/Control.UIManager.js

```diff
@@ -42,13 +42,20 @@ L.Control.UIManager = L.Control.extend({
 
 	initializeBasicUI: function() {
 		var enableNotebookbar = window.userInterfaceMode === 'notebookbar';
+		var that = this;
 
 		if (window.mode.isMobile() || !enableNotebookbar) {
 			var menubar = L.control.menubar();
 			this.map.menubar = menubar;
 			this.map.addControl(menubar);
 		}
 
+		if (window.mode.isMobile()) {
+			$('#toolbar-mobile-back').on('click', function() {
```

Comment:
it would be good later move this to Control.MobileTopBar.js file to keep related things together. you can access uiManager by: `map.uiManager`

Reply:
I will, thank you :)

Reply:
I just checked that Control.MobileTopBar.js is initialized after docloaded event so we cannot move the logic there. The idea is to have this before docloaded

Reply:
ah, ok